### ISSUE TITLE
confused about `Wa` in MLPEncoder

### ISSUE THREAD

**Issue by Lintianqianjin** [fishmoon1234/DAG-GNN#18]
`logits = torch.matmul(adj_Aforz, x+self.Wa) - self.Wa`
what is Wa? It seems that Wa doesn't appear in the paper.
